tests/test_other.py

```diff
@@ -7285,10 +7285,12 @@ def test_multiple_o_files(self):
     assert os.path.isfile('b.js')
     assert not os.path.isfile('a.js')
 
-  # Tests that Emscripten-provided header files can be cleanly included in C code
+  # Tests that Emscripten-provided header files can be cleanly included in C code without
+  # any errors or warnings
   @is_slow_test
   def test_include_system_header_in_c(self):
-    for std in [[], ['-std=c89']]: # Test oldest C standard, and the default C standard
+    # Test oldest C standard, and the default C standard
+    for std in [[], ['-std=c89']]:
```

Comment:
It wouldn't hurt to throw in '`-pedantic', '-Wall', '-Werror'` here, too, to get wider coverage.

Reply:
I add these blow... these are just the *additional* ones on top of those base ones.